src/app/shared/pipes/reaction-message.pipe.ts

```diff
@@ -0,0 +1,27 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { MariosReaction } from 'src/app/core/enums/marios-reaction.enum';
+
+@Pipe({
+  name: 'reactionMessage',
+})
+export class ReactionMessagePipe implements PipeTransform {
+  transform(reaction: MariosReaction): string {
+    console.log(reaction)
```

Comment:
I think it's not needed

Reply:
9650be9c79a7ff81a2ecf32d13b2a6e554778ef6